joomla/joomla-cms
# [4.0] Namespace com_associations

Pull Request for Issue # .

### Summary of Changes
As title says, this PR converts com_associations to namespace MVC for Joomla 4.

### Testing Instructions
1. Setup a multilingual latest using Joomla 4.0-dev branch
2. Test and make sure Multilingual Associations works as expected.
